core/opendaq/device/include/opendaq/device_impl.h

```diff
@@ -1220,14 +1146,10 @@ void GenericDevice<TInterface, Interfaces...>::deserializeCustomObjectValues(con
         deviceInfo.freeze();
     }
 
-    if (serializedObject.hasKey("domainOrigin"))
-        domainOrigin = serializedObject.readString("domainOrigin");
-
-    if (serializedObject.hasKey("domainResolution"))
-        domainTickResolution = serializedObject.readObject("domainResolution");
-
-    if (serializedObject.hasKey("domainUnit"))
-        domainUnit = serializedObject.readObject("domainUnit");
+    if (serializedObject.hasKey("deviceDomain"))
+    {
+        deviceDomain = serializedObject.readObject("deviceDomain");
+    }
 
     this->template deserializeDefaultFolder<IComponent>(serializedObject, context, factoryCallback, ioFolder, "IO");
     this->template deserializeDefaultFolder<IDevice>(serializedObject, context, factoryCallback, devices, "Dev");
```

Comment:
"deviceUnit" (which was wrong anyway, should have been "domainUni") and "deviceResolution" can be removed from ignoredKeys.